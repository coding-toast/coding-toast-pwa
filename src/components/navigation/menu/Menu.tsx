import * as React from 'react'

const MenuLinkStyles = {
  color: 'white',
  textDecoration: 'none',
  fontSize: '24px',
  padding: '10px'
}

const Menu: React.FC = () => (
  <div>
    <div id="menuLinks">
      <a href="/" style={MenuLinkStyles}>
        About
      </a>
      <a href="/" style={MenuLinkStyles}>
        Contact
      </a>
      <a href="/" style={MenuLinkStyles}>
        Login
      </a>
    </div>
  </div>
)

export default Menu
