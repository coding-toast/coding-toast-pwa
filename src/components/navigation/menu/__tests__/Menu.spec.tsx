import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Menu from '../Menu'

describe('<Menu />', () => {
  it('will render correctly', () => {
    const component = shallow(<Menu />)
    expect(toJson(component)).toMatchSnapshot()
  })
})
