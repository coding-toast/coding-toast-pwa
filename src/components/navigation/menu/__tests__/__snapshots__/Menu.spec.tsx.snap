// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Menu /> will render correctly 1`] = `
<div
  id="menuLinks"
>
  <a
    href="/"
    style={
      Object {
        "color": "white",
        "fontSize": "24px",
        "padding": "10px",
        "textDecoration": "none",
      }
    }
  >
    About
  </a>
  <a
    href="/"
    style={
      Object {
        "color": "white",
        "fontSize": "24px",
        "padding": "10px",
        "textDecoration": "none",
      }
    }
  >
    Contact
  </a>
  <a
    href="/"
    style={
      Object {
        "color": "white",
        "fontSize": "24px",
        "padding": "10px",
        "textDecoration": "none",
      }
    }
  >
    Login
  </a>
</div>
`;
